src/shared/components/signUpLink/signUpLink.js

```diff
@@ -1,17 +1,38 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import styles from './signUpLink.css';
 
-const SignUpLink = ({ text }) => (
-  <Link to="/join">{text}</Link>
-);
+class SignUpLink extends Component {
+  state = {
+    btnText: '',
+    paragraph: ''
+  }
+  render() {
+    return (
+      <div className="signin-container">
+        <p>Are you ready to deploy your future?  Join Operation Code
+            today and launch your career in software development.
+            Once you complete the form below you&#8217;ll be invited
+            to join our team on Slack and the <a href="https://community.operationcode.org/" target="_blank" rel="noopener noreferrer">forums</a>.  Make sure you stop in and say hi!
+        </p>
+        <Link className={styles.btnRed} to="/join">{this.props.btnText}</Link>
```

Comment:
Our poor documentation (including outlining the shared components you have available) meant that you had no idea we had a shared component for this precise functionality (LinkButton). 

There was also no need for us to be using a Pure Component or state, so I ditched those as well. The prop being passed to this component seemed unnecessary as well - this button will only ever be labeled "Sign Up".